src/Illuminate/Support/Str.php

```diff
@@ -279,14 +279,19 @@ public static function singular($value)
 	/**
 	 * Generate a URL friendly "slug" from a given string.
 	 *
-	 * @param  string  $title
-	 * @param  string  $separator
+	 * @param  string $title
+	 * @param  string $separator
+	 * @param array $replacements
 	 * @return string
 	 */
-	public static function slug($title, $separator = '-')
+	public static function slug($title, $separator = '-', $replacements = array())
 	{
 		$title = static::ascii($title);
 
+		$replacements['&'] = isset($replacements['&']) ? $replacements['&'] : 'and';
```

Comment:
I would probably prefer moving this to function argument to make it disableable
`public static function slug($title, $separator = '-', $replacements = ['&' => 'and'])`